keras_cv/models/mobilenet_v3.py

```diff
@@ -72,33 +72,33 @@
                 of filters in each layer.
             - If `alpha` = 1, default number of filters from the paper
                 are used at each layer.
-        minimalistic: in addition to large and small models this module also
-            contains so-called minimalistic models, these models have the same
+        minimalistic: in addition to large and small models, this module also
+            contains so-called minimalistic models; these models have the same
             per-layer dimensions characteristic as MobilenetV3 however, they don't
             utilize any of the advanced blocks (squeeze-and-excite units, hard-swish,
             and 5x5 convolutions). While these models are less efficient on CPU, they
             are much more performant on GPU/DSP.
         dropout_rate: a float between 0 and 1 denoting the fraction of input units to
             drop, defaults to 0.2.
         classifier_activation: the activation function to use, defaults to softmax.
-        name: (Optional) name to pass to the model. Defaults to "{name}".
+        name: string, optional name to pass to the model, defaults to "{name}".
 
     Returns:
         A `keras.Model` instance.
 """
 
 
-def depth(x, divisor=8, min_value=None):
-    """Ensure that all layers have a channel number that is divisble by the `divisor`.
+def apply_depth(x, divisor=8, min_value=None):
```

Comment:
nit: this isn't a function on a Tensor, but a helper method to get the number of channels for a convolutional layer, so it should just be `depth` instead of `apply_depth`